network-api/networkapi/templates/partials/multipage-nav-mobile.html

```diff
@@ -0,0 +1,7 @@
+<div class="hidden-md-up {{wrapper_classes}}" id="multipage-nav-mobile">
```

Comment:
I'd probably put this in `network-api/networkapi/fellowships/templates/fellowships/` (which django understands as the "local" template directory for an app. And yes, it's `{appname}/templates/{appname}` because technically you can also do things like `{appname}/templates/admin` for overriding the django admin interface, etc. Bit silly, but eh) or maybe even in `network-api/networkapi/fellowships/templates/fellowships/fragments`.

Is this the right PR to make sure felloships templates are stored locally to that app, @alanmoo?

Reply:
then let's keep it in the shared templates dir.